frequency: 1
# 0 = default(list all), 1 = whitelist, 2 = blacklist
filter_type: 1

rosout:
  values:
    - name
    - cmdline
    - cpu_percent
    - memory_percent
node_ressource_monitor:
  values:
    - exe
    - cpu_percent
    - memory_percent
